babel.config.extensions.js

```diff
@@ -0,0 +1,58 @@
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				useBuiltIns: 'entry',
+				corejs: 2,
```

Comment:
this is, I think, what is throwing the error Travis saw.

Reply:
I just bumped some Babel package versions to fix this :crossed_fingers: